src/app/main-api-handler.ts

```diff
@@ -291,6 +291,14 @@ ipcMain.on(
             ?.webContents.setZoomFactor(arg.zoomLevel as number);
         }
         break;
+      case apiCmds.aboutAppClipBoardData:
```

Comment:
[Question] What is the purpose of "aboutAppClipBoardData"?

Reply:
This is where we collect different config file data from customers